pkg/kyma/cmd/test/status/cmd.go

```diff
@@ -0,0 +1,118 @@
+package status
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	oct "github.com/kyma-incubator/octopus/pkg/apis/testing/v1alpha1"
+	"github.com/kyma-project/cli/pkg/kyma/cmd/test"
+	"github.com/kyma-project/cli/pkg/kyma/cmd/test/client"
+	"github.com/kyma-project/cli/pkg/kyma/core"
+	"github.com/spf13/cobra"
+)
+
+type command struct {
+	opts *options
+	core.Command
+}
+
+func NewCmd(o *options) *cobra.Command {
+	cmd := command{
+		Command: core.Command{Options: o.Options},
+		opts:    o,
+	}
+
+	cobraCmd := &cobra.Command{
+		Use:     "status",
+		Short:   "Status of tests on a running Kyma cluster",
+		Long:    `Status of tests on a running Kyma cluster`,
+		RunE:    func(_ *cobra.Command, args []string) error { return cmd.Run(args) },
+		Aliases: []string{"s"},
+	}
+
+	//cobraCmd.Flags().BoolVarP(&o.Wait, "wait", "w", false, "Wait for test execution to finish")
+	//cobraCmd.Flags().StringVarP(&o.Logs, "logs", "l", "", "Test name to show logs")
+	cobraCmd.Flags().BoolVarP(&o.Jsn, "raw", "r", false,
+		"Print test status in raw json format")
+	return cobraCmd
+}
+
+func (cmd *command) Run(args []string) error {
+	cli, err := client.NewTestRESTClient(10 * time.Second)
+	if err != nil {
+		return fmt.Errorf("unable to create test REST client. E: %s", err)
```

Comment:
That "E:" in all the error messages, I think that is not a known convention and hard to guess when reading it in the CLI output.

Reply:
Fixed